app/core/Tracker2/BaseTracker.js

```diff
@@ -120,8 +132,8 @@ export default class BaseTracker {
     const finishInitialization = (result) => {
       this.isInitialized = result
 
-      delete this.onInitializeSuccess
```

Comment:
Interested in what difference this makes with the `() => {}` assignment?

Reply:
Good question - I'll add clarification as code comment.